apps/backend/src/modules/vendors/libs/types/vendor-service.type.ts

```diff
@@ -0,0 +1,22 @@
+import { CourseDto } from "shared";
```

Comment:
```suggestion
import { type CourseDto } from "shared";
```
And maybe import should be from `types.ts`, not from `shared`